db/schema.rb

```diff
@@ -803,6 +803,7 @@
     t.bigint "assigned_by_id"
     t.index ["assigned_by_id"], name: "index_user_projects_on_assigned_by_id"
     t.index ["project_id"], name: "index_user_projects_on_project_id"
+    t.index ["user_id", "project_id"], name: "index_user_projects_on_user_id_and_project_id", unique: true
```

Comment:
Are you adding indexes without migration?

Reply:
Sh*t, I don't know where this comes from 😨

Reply:
Looks like this is from this [PR](https://github.com/biosistemika/scinote-web/pull/1532)
But don't know how this happend, because looks like @radiokills pushed `schema.rb`. ?